src/server/status/index.js

```diff
@@ -29,7 +34,8 @@ export default function (kbnServer, server, config) {
           build_snapshot: matchSnapshot.test(config.get('pkg.version'))
         },
         status: kbnServer.status.toJSON(),
-        metrics: kbnServer.metrics
+        metrics: kbnServer.metrics,
+        stats,
```

Comment:
Whether we use `/_kibana/*` or `/api/*` does not make a big difference to me other than to note that the rest of the stack has gone this direction. I do like our `/api` prefix particularly because it means we can setup request filters and such based on the flow of the URL itself. So perhaps `/api/_kibana/stats` might even be a better combo since the non-`/api` URLs have a very different meaning.